src/services/baseService.ts

```diff
@@ -11,15 +12,15 @@ export abstract class BaseService {
   protected resourceGroup: string;
   protected deploymentName: string;
 
-  public constructor(protected serverless: Serverless, protected options: Serverless.Options) {
+  public constructor(protected serverless: Serverless, protected options: Serverless.Options, authenticate = true) {
```

Comment:
Added this for services like `ConverterService` or `OfflineService` that could benefit from shared service code, but that don't need to authenticate with Azure

Reply:
Update: `ConverterService` no longer part of this. Using normal packaging process to build `function.json` files